README.md

```diff
@@ -70,7 +70,7 @@ When in doubt, ask a [core team member](#core-team)! You can mention us in any i
 
 **Refactoring** code, e.g. improving the code without modifying the behavior is an area that can probably be done based on intuition and may not require much communication to be merged.
 
-**Fixing bugs** may also not require a lot of communication, but the more the better. Please surround bug fixes with ample tests. Bugs are magnets for other bugs. Write tests near bugs!
+**Fixing bugs** may also not require a lot of communication, but the more the better. Please surround bug fixes with sample tests. Bugs are magnets for other bugs. Write tests near bugs!
```

Comment:
Maybe they meant "ample" as in [generous, more than adequate](https://www.merriam-webster.com/dictionary/ample), this might not be a typo after all, who knows 😄